packages/styling/src/interfaces/ISemanticTextColors.ts

```diff
@@ -64,14 +64,20 @@ export interface ISemanticTextColors {
   //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
 
   /**
-   * The default color of error text, used on bodyBackground.
+   * The default color of text indicating error on bodyBackground.
    */
   errorText: string;
+
   /**
-   * The color of text on errorBackground, warningBackground, blockingBackground, or successBackground.
+   * The color of text indicating warning on errorBackground, warningBackground, blockingBackground, or successBackground.
```

Comment:
this does not indicate warnings, it's just the default text color on these backgrounds, like body text basically. it's unfortunately named, couldn't think of a better name at the time =(